Q2ShawnCampbell.md

```diff
@@ -0,0 +1,148 @@
+# Quiz #2
+
+## Instructions
+
+1. Fork this repo
+2. Clone your fork
+3. Fill in your answers by writing in the appropriate area, or placing an 'x' in
+the square brackets (for multiple-choice questions).
+4. Add/Commit/Push your changes to Github.
+5. Open a pull request.
+
+## CSS
+
+### Question #1
+
+Describe the purpose of a clearfix in CSS, and give an example of how to do it.
+
+Your Answer:
+```The clearfix is a CSS method used to make it easier to format a group of
+images.
+```
+
+### Question #2
+
+What does the following selector do?  `ul.dropdown > li`?
+
+Select 1:
+```
+[] Selects all li's which are directly inside a ul of class dropdown (children)
+[X] Selects all li's which are anywhere inside a ul of class dropdown (any descendant)
+[] Selects all ul's of class dropdown, as well as the children elements that are li's
+[] Selects all ul's of class dropdown, only if their children are exclusively li's
+```
+
```

Comment:
Close. `>` actually means you are selecting the direct children of an element. So the correct answer is the first choice.